lib/std/os/linux.zig

```diff
@@ -4525,6 +4537,11 @@ pub const clockid_t = enum(u32) {
     _,
 };
 
+pub const TIMER = packed struct(u32) {
+    ABSTIME: bool,
+    _: u31 = undefined,
```

Comment:
This is usually required to be future-proof:

```suggestion
    _: u31 = 0,
```